packages/argos-cli/src/index.js

```diff
@@ -0,0 +1,7 @@
+import program from 'commander';
+
+program.parse(process.argv);
```

Comment:
I'm asking because it's what we are doing the test cli for material. Avoiding running command when not argument are provided.